README.md

```diff
@@ -244,6 +275,19 @@ Manifest is loaded as **template**, so you can use variables, Go **range** and *
             ]
         },
 
+        // Cookies will be under this key, in a map name => cookie
+        "cookie": {
+            "cookie1": {
+                "name": "cookie1",
+                "value": "val1",
+                "path": "/"
```

Comment:
Is this list complete? I think a least expires should be in there too...

Reply:
With name and val it is complete, everything else is optional, but I can add more